ml_enabler/aggregators/BuildingAggregator.py

```diff
@@ -0,0 +1,37 @@
+from ml_enabler.aggregators.BaseAggregator import BaseAggregator
+from ml_enabler.utils import get_building_area, get_tile_center
+import functools
+import mercantile
+import aiohttp
+import asyncio
+import json
+
+
+class BuildingAggregator(BaseAggregator):
+
+    async def aggregate(self):
+        conn = aiohttp.TCPConnector(limit=3)
```

Comment:
@geohacker we should eventually make this a param and set it in the base class and get it from `self.concurrency` or so. Just dropping this here to remember to change this when we add `concurrency` as a param to the aggregate script.